README.md

```diff
@@ -25,12 +26,67 @@ First you need to create a `vars.env` file with `PORT=3100` (Or any port you lik
 3. Run `npm start` to start the server
 4. You can now go to [`http://localhost:3100/`](http://localhost:3100/)
 
+# Concept
+The concept of this product is to have an introductory to the story of the Ceuvel. By telling the story you can't see the effect that it has on the overall energy consumption or energy output. By making a nice data visualisation of every data output on a screen will not do the **awesome** story of the Ceuvel justice.
+
+Our concept is a Augmented Reality (AR) tour that you can follow on the awesome ground of the Ceuvel. There will be AR stickers on the ground that you can scan with the Website. What you will see after scanning is live data graph in AR with this you can actually see what it going on this particular Boat. You will find a bit of text in AR that will explain what you see and what this will benefit.
```

Comment:
scanning is a live data graph in AR

which will show you what is 

text in AR which will explain what you see and what benefit this brings